src/gc/impl/conservative/gc.d

```diff
@@ -2854,15 +2854,16 @@ struct Gcx
 
     void scanBackground() nothrow
     {
-        version(linux) {
+        version(linux)
+        {
             import core.sys.posix.signal;
 
             // block all signals, so that scanThread will not receive a signal not intended for him.
             // see https://issues.dlang.org/show_bug.cgi?id=20256
             sigset_t new_mask;
             sigfillset(&new_mask);
-			int sigmask_rc = pthread_sigmask(SIG_BLOCK, &new_mask, null);
-			assert(sigmask_rc == 0, "failed to set up GC scan thread sigmask");
+            int sigmask_rc = pthread_sigmask(SIG_BLOCK, &new_mask, null);
+            assert(sigmask_rc == 0, "failed to set up GC scan thread sigmask");
 		}
         while (!stopGC)
```

Comment:
These two ought to line up. I think you need to indent the entire `version(linux)` block by two spaces.